frontend/src/components/entryboard/TeamInfo.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+import { ref, reactive, onMounted } from 'vue';
+
+//TODO: common location for interfaces.
```

Comment:
This is good for leaving a TODO here. There might be a conversation down the road with all of the other teams later to define an SDK that can be used across multiple projects, but I think that is outside of the scope of MVP for us.